File: Java/POO 2/Vehiculo/Coche.java
public class Coche extends Vehiculo{
  
  //Cilindrada del coche
  private int cilindrada;
  
  public Coche(int c) {
    super();
    this.cilindrada=c;
  }
  
  public void quemaRueda() {
    System.out.println("Estoy quemando rueda");
  }
}

File: Python/POO 2/Vehiculo/Coche.py
'''
Created on 6 feb. 2019

@author: Cristina Prieto Jalao
'''
from Vehiculo import Vehiculo  #Importo la clase Vehiculo

class Coche(Vehiculo):  #Subclase de Vehiculo
    #Constructor
    def __init__(self, km, c):
        Vehiculo.__init__(self, km)
        self.cilindrada=c
        
    def quemaRueda(self):
        print("Estoy quemando rueda")